manifests/worker.pp

```diff
@@ -51,6 +51,18 @@
 #   Example: 'rdo-trunk@rdoproject.org'
 #   Defaults to undef
 # 
+# [*rsyncdest*]
+#   (optional) name or ip address for server where builtdir and reports are copied
+#     after built is successfull.
+#   Example: 'backupserver.example.com'
```

Comment:
This is not a good example. Let's use a better one, like root@backupserver.example.com:/backupdir